src/include/distributed/query_utils.h

```diff
@@ -0,0 +1,40 @@
+/*-------------------------------------------------------------------------
+ *
+ * query_utils.h
+ *
+ * Declarations for query-walker utility functions and related types.
+ *
+ * Copyright (c) 2019, Citus Data, Inc.
+ *
+ ***-------------------------------------------------------------------------
+ */
+#ifndef QUERY_UTILS_H
+#define QUERY_UTILS_H
+
+#include "postgres.h"
+#include "nodes/pg_list.h"
+
+/* Enum to define execution flow of ExtractRangeTableList */
+typedef enum ExtractMode
+{
+	EXT_RELATION_ENTRIES,
+	EXT_INDEX_ENTRIES,
+	EXT_ALL_ENTRIES
```

Comment:
We try to avoid acronyms (We as in the developers of this codebase, not Microsoft ;). 

So maybe `EXTRACT_` instead of `EXT_`